admin/services/data-access/pupil-attendance.data.service.js

```diff
@@ -70,5 +70,27 @@ pupilAttendanceDataService.findByPupilIds = async (ids) => {
   const sql = [select, whereClause].join(' ')
   return sqlService.query(sql, params)
 }
+/**
+ * Returns pupil attendance based on pupil id
+ * @param pupilId
+ * @return {Object}
+ */
+pupilAttendanceDataService.findByPupilId = async (pupilId) => {
```

Comment:
Should be called `findOneById` please
This whole method can be replaced by a call to `sqlService.findOneById()`

Reply:
@jon-shipley updated